diff --git a/Kptfile b/Kptfile
index d43e47a..7a7c0cb 100644
--- a/Kptfile
+++ b/Kptfile
@@ -6,7 +6,12 @@ pipeline:
   mutators:
     - image: ghcr.io/kptdev/krm-functions-catalog/apply-setters:latest
       configMap:
-        name: my-new-map
         env: |
           - prod
           - stage
+        name: my-new-map
+status:
+  conditions:
+    - type: Rendered
+      status: "True"
+      reason: RenderSuccess
