diff --git a/Kptfile b/Kptfile
index 0d32b6f..b9e2ee6 100644
--- a/Kptfile
+++ b/Kptfile
@@ -7,3 +7,41 @@ metadata:
 pipeline:
   validators:
     - image: ghcr.io/kptdev/krm-functions-catalog/gatekeeper:latest
+status:
+  conditions:
+    - type: Rendered
+      status: "False"
+      reason: RenderFailed
+      message: |-
+        pkg.render: pkg .:
+        	pipeline.run: already handled error
+  renderStatus:
+    validationSteps:
+      - image: ghcr.io/kptdev/krm-functions-catalog/gatekeeper:latest
+        stderr: |-
+          [error] apps/v1/Deployment/nginx-deploy : Containers must not run as root
+          violatedConstraint: disallowroot
+        exitCode: 1
+        results:
+          - message: |-
+              Containers must not run as root
+              violatedConstraint: disallowroot
+            severity: error
+            resourceRef:
+              apiVersion: apps/v1
+              kind: Deployment
+              name: nginx-deploy
+            file:
+              path: deployment.yaml
+        errorResults:
+          - message: |-
+              Containers must not run as root
+              violatedConstraint: disallowroot
+            severity: error
+            resourceRef:
+              apiVersion: apps/v1
+              kind: Deployment
+              name: nginx-deploy
+            file:
+              path: deployment.yaml
+    errorSummary: 'ghcr.io/kptdev/krm-functions-catalog/gatekeeper:latest: exit code 1'
