diff --git a/Kptfile b/Kptfile
index 43a422f..a407221 100644
--- a/Kptfile
+++ b/Kptfile
@@ -8,3 +8,15 @@ pipeline:
   mutators:
     - image: ghcr.io/kptdev/krm-functions-catalog/set-image:latest
       configPath: fn-config.yaml
+status:
+  conditions:
+    - type: Rendered
+      status: "True"
+      reason: RenderSuccess
+  renderStatus:
+    mutationSteps:
+      - image: ghcr.io/kptdev/krm-functions-catalog/set-image:latest
+        exitCode: 0
+        results:
+          - message: 'summary: updated a total of 2 image(s)'
+            severity: info
diff --git a/resources.yaml b/resources.yaml
index c15f5ab..2f33677 100644
--- a/resources.yaml
+++ b/resources.yaml
@@ -5,7 +5,7 @@ metadata:
 spec:
   containers:
   - name: server
-    image: nginx:1.20.2
+    image: bitnami/nginx:1.21.4
   - name: store
     image: postgres:14.1
 ---
@@ -17,5 +17,5 @@ metadata:
 spec:
   manifest:
     images:
-    - image: nginx:1.20.2
+    - image: bitnami/nginx:1.21.4
     - image: postgres:14.1
