diff --git a/Kptfile b/Kptfile
index 5aa759e..8ac91d5 100644
--- a/Kptfile
+++ b/Kptfile
@@ -11,3 +11,38 @@ pipeline:
       selectors:
         - kind: Deployment
         - kind: MyKind
+status:
+  conditions:
+    - type: Rendered
+      status: "True"
+      reason: RenderSuccess
+  renderStatus:
+    mutationSteps:
+      - image: ghcr.io/kptdev/krm-functions-catalog/create-setters:latest
+        exitCode: 0
+        results:
+          - message: 'Added line comment "kpt-set: /tmp/kpt-pipeline-e2e-39110${nginx-replicas}1686/create-setters-simple" for field with value "/tmp/kpt-pipeline-e2e-3911041686/create-setters-simple"'
+            field:
+              path: metadata.annotations.internal.config.kubernetes.io/package-path
+            file:
+              path: resources.yaml
+          - message: 'Added line comment "kpt-set: ${nginx-replicas}" for field with value "4"'
+            field:
+              path: spec.replicas
+            file:
+              path: resources.yaml
+          - message: 'Added line comment "kpt-set: nginx:${tag}" for field with value "nginx:1.16.1"'
+            field:
+              path: spec.template.spec.containers[0].image
+            file:
+              path: resources.yaml
+          - message: 'Added line comment "kpt-set: ${env}" for field with value "[dev stage]"'
+            field:
+              path: environments
+            file:
+              path: resources.yaml
+          - message: 'Added line comment "kpt-set: /tmp/kpt-pipeline-e2e-39110${nginx-replicas}1686/create-setters-simple" for field with value "/tmp/kpt-pipeline-e2e-3911041686/create-setters-simple"'
+            field:
+              path: metadata.annotations.internal.config.kubernetes.io/package-path
+            file:
+              path: resources.yaml
diff --git a/resources.yaml b/resources.yaml
index 716ecb9..b521905 100644
--- a/resources.yaml
+++ b/resources.yaml
@@ -3,7 +3,7 @@ kind: Deployment
 metadata:
   name: my-nginx
 spec:
-  replicas: 4
+  replicas: 4 # kpt-set: ${nginx-replicas}
   selector:
     matchLabels:
       app: nginx
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
         - name: nginx
-          image: "nginx:1.16.1"
+          image: "nginx:1.16.1" # kpt-set: nginx:${tag}
           ports:
             - protocol: TCP
               containerPort: 80
@@ -23,6 +23,6 @@ apiVersion: v1
 kind: MyKind
 metadata:
   name: foo
-environments:
+environments: # kpt-set: ${env}
   - dev
   - stage
