diff --git a/Kptfile b/Kptfile
index e9b883d..0594f99 100644
--- a/Kptfile
+++ b/Kptfile
@@ -6,3 +6,11 @@ pipeline:
   mutators:
     - image: ghcr.io/kptdev/krm-functions-catalog/starlark:latest
       configPath: fn-config.yaml
+status:
+  conditions:
+    - type: Rendered
+      status: "False"
+      reason: RenderFailed
+      message: |-
+        pkg.render: pkg .:
+        	pipeline.run: already handled error
