diff --git a/Kptfile b/Kptfile
index f3843da..334d29f 100644
--- a/Kptfile
+++ b/Kptfile
@@ -8,3 +8,12 @@ pipeline:
   mutators:
     - image: ghcr.io/kptdev/krm-functions-catalog/apply-replacements:latest
       configPath: replacements.yaml
+status:
+  conditions:
+    - type: Rendered
+      status: "True"
+      reason: RenderSuccess
+  renderStatus:
+    mutationSteps:
+      - image: ghcr.io/kptdev/krm-functions-catalog/apply-replacements:latest
+        exitCode: 0
diff --git a/job.yaml b/job.yaml
index 4584bb9..66f3ebe 100644
--- a/job.yaml
+++ b/job.yaml
@@ -2,3 +2,10 @@ apiVersion: batch/v1
 kind: Job
 metadata:
   name: hello
+spec:
+  template:
+    spec:
+      containers:
+      - image: busybox
+        name: myapp-container
+      restartPolicy: OnFailure
