diff --git a/Kptfile b/Kptfile
index 6f2b190..9672642 100644
--- a/Kptfile
+++ b/Kptfile
@@ -8,3 +8,23 @@ pipeline:
   validators:
     - image: ghcr.io/kptdev/krm-functions-catalog/starlark:latest
       configPath: fn-config.yaml
+status:
+  conditions:
+    - type: Rendered
+      status: "False"
+      reason: RenderFailed
+      message: |-
+        pkg.render: pkg .:
+        	pipeline.run: already handled error
+  renderStatus:
+    validationSteps:
+      - image: ghcr.io/kptdev/krm-functions-catalog/starlark:latest
+        stderr: 'failed to evaluate function: error: function failure'
+        exitCode: 1
+        results:
+          - message: 'fail: it is prohibited to have private key in a configmap'
+            severity: error
+        errorResults:
+          - message: 'fail: it is prohibited to have private key in a configmap'
+            severity: error
+    errorSummary: 'ghcr.io/kptdev/krm-functions-catalog/starlark:latest: exit code 1'
