diff --git a/Kptfile b/Kptfile
index 28af78d..4ab6694 100644
--- a/Kptfile
+++ b/Kptfile
@@ -6,3 +6,23 @@ pipeline:
   mutators:
     - image: ghcr.io/kptdev/krm-functions-catalog/set-annotations:latest
       configPath: fn-config.yaml
+status:
+  conditions:
+    - type: Rendered
+      status: "False"
+      reason: RenderFailed
+      message: |-
+        pkg.render: pkg .:
+        	pipeline.run: already handled error
+  renderStatus:
+    mutationSteps:
+      - image: ghcr.io/kptdev/krm-functions-catalog/set-annotations:latest
+        stderr: '[error]: failed to configure function: input annotation list cannot be empty'
+        exitCode: 1
+        results:
+          - message: 'failed to configure function: input annotation list cannot be empty'
+            severity: error
+        errorResults:
+          - message: 'failed to configure function: input annotation list cannot be empty'
+            severity: error
+    errorSummary: 'ghcr.io/kptdev/krm-functions-catalog/set-annotations:latest: exit code 1'
