diff --git a/Kptfile b/Kptfile
index bc7986a..3ecce01 100644
--- a/Kptfile
+++ b/Kptfile
@@ -9,3 +9,23 @@ pipeline:
     - image: ghcr.io/kptdev/krm-functions-catalog/set-enforcement-action:latest
       configMap:
         enforcementAction: dryrun
+status:
+  conditions:
+    - type: Rendered
+      status: "True"
+      reason: RenderSuccess
+  renderStatus:
+    mutationSteps:
+      - image: ghcr.io/kptdev/krm-functions-catalog/set-enforcement-action:latest
+        exitCode: 0
+        results:
+          - message: 'Number of policies set to [dryrun]: 2'
+            severity: info
+          - message: 'Policy name: [restrict-clusteradmin-rolebindings]'
+            severity: info
+            file:
+              path: resources.yaml
+          - message: 'Policy name: [prohibit-role-wildcard-access]'
+            severity: info
+            file:
+              path: resources.yaml
diff --git a/resources.yaml b/resources.yaml
index 3e82c66..c15aa3c 100644
--- a/resources.yaml
+++ b/resources.yaml
@@ -7,7 +7,7 @@ metadata: # kpt-merge: /restrict-clusteradmin-rolebindings
     description: "Restricts use of the cluster-admin role."
     bundles.validator.forsetisecurity.org/cis-k8s-v1.5.1: 5.1.1
 spec:
-  enforcementAction: deny
+  enforcementAction: dryrun
   parameters:
     restrictedRole:
       apiGroup: "rbac.authorization.k8s.io"
@@ -27,4 +27,4 @@ metadata: # kpt-merge: /prohibit-role-wildcard-access
     description: "Restricts use of wildcards in Roles and ClusterRoles."
     bundles.validator.forsetisecurity.org/cis-k8s-v1.5.1: 5.1.3
 spec:
-  enforcementAction: deny
+  enforcementAction: dryrun
