diff --git a/Kptfile b/Kptfile
index 152ea13..d2c2af5 100644
--- a/Kptfile
+++ b/Kptfile
@@ -8,3 +8,53 @@ pipeline:
   mutators:
     - image: ghcr.io/kptdev/krm-functions-catalog/apply-setters:latest
       configPath: setters.yaml
+status:
+  conditions:
+    - type: Rendered
+      status: "True"
+      reason: RenderSuccess
+  renderStatus:
+    mutationSteps:
+      - image: ghcr.io/kptdev/krm-functions-catalog/apply-setters:latest
+        exitCode: 0
+        results:
+          - message: set field value to "3"
+            severity: info
+            resourceRef:
+              apiVersion: apps/v1
+              kind: Deployment
+              name: my-nginx
+            field:
+              path: spec.replicas
+              currentValue: "4"
+              proposedValue: "3"
+            file:
+              path: resources.yaml
+          - message: set field value to "nginx:1.16.2"
+            severity: info
+            resourceRef:
+              apiVersion: apps/v1
+              kind: Deployment
+              name: my-nginx
+            field:
+              path: spec.template.spec.containers[0].image
+              currentValue: nginx:1.16.1
+              proposedValue: nginx:1.16.2
+            file:
+              path: resources.yaml
+          - message: set field value to "- prod\n- dev"
+            severity: info
+            resourceRef:
+              apiVersion: v1
+              kind: MyKind
+              name: foo
+            field:
+              path: environments
+              currentValue: |-
+                - dev
+                - stage
+              proposedValue: |-
+                - prod
+                - dev
+            file:
+              path: resources.yaml
diff --git a/resources.yaml b/resources.yaml
index 3dde2b6..47b7163 100644
--- a/resources.yaml
+++ b/resources.yaml
@@ -3,7 +3,7 @@ kind: Deployment
@@ -20,18 +20,20 @@
            ports:
              - protocol: TCP
                containerPort: 80
-@@ -24,5 +24,5 @@ kind: MyKind
+@@ -24,13 +24,13 @@ kind: MyKind
  metadata:
    name: foo
  environments: # kpt-set: ${env}
 +  - prod
    - dev
 -  - stage
-@@ -24,5 +24,5 @@ kind: MyKind
+ ---
+ apiVersion: v1
+ kind: MyKind
  metadata:
    name: bar
- environments: # kpt-set: ${env}
-+  - key: some-other-key
-     value: some-other-value
+ environments: # kpt-set: ${nested-env}
 -  - key: some-key
-     value: some-value
\ No newline at end of file
+-    value: some-value
++  - key: some-other-key
++    value: some-other-value