diff --git a/Kptfile b/Kptfile
index 152ea13..6eaff9b 100644
--- a/Kptfile
+++ b/Kptfile
@@ -8,3 +8,28 @@ pipeline:
   mutators:
     - image: ghcr.io/kptdev/krm-functions-catalog/apply-setters:latest
       configPath: setters.yaml
+status:
+  conditions:
+    - type: Rendered
+      status: "True"
+      reason: RenderSuccess
+  renderStatus:
+    mutationSteps:
+      - image: ghcr.io/kptdev/krm-functions-catalog/apply-setters:latest
+        exitCode: 0
+        results:
+          - message: set field value to "3"
+            field:
+              path: spec.replicas
+            file:
+              path: resources.yaml
+          - message: set field value to "nginx:1.16.2"
+            field:
+              path: spec.template.spec.containers[0].image
+            file:
+              path: resources.yaml
+          - message: set field value to "- prod\n- dev"
+            field:
+              path: environments
+            file:
+              path: resources.yaml
diff --git a/resources.yaml b/resources.yaml
index b521905..0d10662 100644
--- a/resources.yaml
+++ b/resources.yaml
@@ -3,7 +3,7 @@ kind: Deployment
 metadata:
   name: my-nginx
 spec:
-  replicas: 4 # kpt-set: ${nginx-replicas}
+  replicas: 3 # kpt-set: ${nginx-replicas}
   selector:
     matchLabels:
       app: nginx
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
         - name: nginx
-          image: "nginx:1.16.1" # kpt-set: nginx:${tag}
+          image: "nginx:1.16.2" # kpt-set: nginx:${tag}
           ports:
             - protocol: TCP
               containerPort: 80
@@ -24,5 +24,5 @@ kind: MyKind
 metadata:
   name: foo
 environments: # kpt-set: ${env}
+  - prod
   - dev
-  - stage
